datafusion/optimizer/src/decorrelate_predicate_subquery.rs

```diff
@@ -296,37 +287,26 @@ fn build_join_top(
 ///
 /// ```text
 /// Projection: t1.id
-///   Filter: t1.id < 0 OR __correlated_sq_1.__exists IS NOT NULL
+///   Filter: t1.id < 0 OR __correlated_sq_1.mark
```

Comment:
Shouldn't this have `mark join` here in the doc?

Reply:
Fixed in https://github.com/apache/datafusion/pull/13134/commits/e1809392649611f24ea112e214e54c2c353d2aa8